src/main/java/me/karakelley/tictactoe/EasyBehavior.java

```diff
@@ -0,0 +1,21 @@
+package me.karakelley.tictactoe;
+
+import java.util.List;
+import java.util.Random;
+
+public class EasyBehavior implements AIBehavior {
+
+  @Override
+  public String move(BoardState boardState) {
```

Comment:
Why would this return a `String` instead of an `Integer`?

Reply:
When the placeMove function receives input it converts it to an integer. I thought it was better to do it in once place, at the very end, instead of converting it to an integer when the computer and human chooses a space. Is it better to convert it right away?